### ISSUE TITLE
Deprecate package?

### ISSUE THREAD

**Issue by jonkoops** [jshttp/methods#29]
Since Node.js 0.10 and lower are essentially no longer used and Browserify largely replaced by tools such as Webpack and Vite it might be time to sunset this package. Perhaps the Express v5 release might be a good time to officially deprecate the package on NPM as well?

**Comment by UlisesGascon** [comment]
So far seems like the package is quite popular https://www.npmjs.com/package/methods
![Screenshot from 2024-10-22 13-12-51](https://github.com/user-attachments/assets/b7466d37-7ba1-4629-810d-2c36f2b591d8)

that being said, we might want to do an effort to release a new version with all the unpublished changes and maybe also taking the opportunity to revisit the CI and the dependencies that we use.

**Comment by jonkoops** [comment]
Indeed. I do wonder what is causing this large amount of downloads. This has to be because popular libraries depend on `methods`. Unfortunately, the analytics on NPM are quite poor and it is not really easy to see which dependents are causing most of the downloads.

**Comment by blakeembrey** [comment]
Sorry, I just saw https://github.com/pillarjs/router/pull/127, thanks! I'm working through my email in reverse chronological order 😄

**Comment by jonkoops** [comment]
Perhaps a notice will suffice for now then. Would you like me to create a PR for it?

**Comment by wesleytodd** [comment]
Yeah I agree that we should not deprecate until 4 is well into EOL, and that we should remove in v5. I am on the fence on the notice unless we make it clear this is not a bad thing that it is used in `express@4` because just like deprecating it causes churn which is unnecessary. That said, once v4 is nearing EOL I would be in favor of a message and then we can deprecate later.

Can we close this or is there more to discuss?

**Comment by UlisesGascon** [comment]
nevermind.. we can close it , I can't not move the issue because is in another org.
